### ISSUE TITLE
[Element Tree] Loading indicator

### ISSUE THREAD

**Issue by markus-moser** [pimcore/studio-ui-bundle#722]
I guess we should do it like in the ant design tree:

https://ant.design/components/tree#tree-demo-dynamic

**Comment by vin0401** [comment]
The loading indicator for folders and the initial loading is solved via: https://github.com/pimcore/studio-ui-bundle/pull/746

But we should also discuss how to deal with the different context menu actions.
For example the deletion of an item invalidates the data of the tree and will force the loading indicator for the parent folder to appear when the deletion succeeds. What is perfectly fine. But we are missing an indication for the time where the deletion itself is still ongoing. Of course this also applies to several other actions. 

@markus-moser @idaiv - Any ideas on this? I guess we need to take a deeper look at all the different behaviors from the context menu actions?

**Comment by markus-moser** [comment]
@vin0401 I think we have something similar in the classic bundle:

![Image](https://github.com/user-attachments/assets/f8722bde-b53e-46a3-9f04-cc13aacf10c8)

In my opinion the new approach should be similiar to this - WDYT?

**Comment by idaiv** [comment]
Hi @markus-moser and @vin0401!
The approach of ant for the tree is perfect! https://ant.design/components/tree#tree-demo-dynamic

> But we should also discuss how to deal with the different context menu actions. For example the deletion of an item invalidates the data of the tree and will force the loading indicator for the parent folder to appear when the deletion succeeds. What is perfectly fine. But we are missing an indication for the time where the deletion itself is still ongoing. Of course this also applies to several other actions.
> 
I would say if the loading indicator in this case is for the whole folder, what we can do to identify the element which deletion is in progress could be to blur it. That way it will have some color separation and blurred or lighter font in general communicates not available/ disabled/ etc. This is great to give a hint for the user initially. We can think of exchanging the element icon with the loading (https://ant.design/components/tree#tree-demo-dynamic) + blur to transparently show that this is not a permanent state but something is happening with this file and therefore its blurred as it is disabled right now. 

WDYT?

**Comment by markus-moser** [comment]
@idaiv Yes, this sounds like a good idea and is a very similar approach to the old UI - for sure we should not style it with the same red strikethrough, the blur sounds like it could work!

**Comment by idaiv** [comment]
Applied for the tags loading too: https://pimcore-my.sharepoint.com/:v:/p/christine_altmann/EXbR8J9OjexCrW43RoVnzegBMu1NG2rdl-ffHEtyszHU7w?e=0jReaW
